README.md

```diff
@@ -1,16 +1,102 @@
-# Title
+# Git Merge Conflicts
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Identify merge conflicts
+- Recognize resolving merge conflicts
+- Recognize reverting merges
 
 ## Introduction
 
-## SWBAT 1
+When working on our own on utilizing a standard git workflow a project, chances
```

Comment:
I think some rewording would flow better here. Maybe something like: "When we're using Git to organize a solo project, there's a low chance of running into code conflicts between branches. However ..."